description.md

```diff
@@ -8,6 +8,70 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Lectures
 
+Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.
+Interaction is (quite obviously) important, but it won't scale to a larger audience and, because the entire communication medium is shared, is limited to on conversation only.
+Labs, for example, allow each participant to focus on one item they could work on.
+Seminars are close to lectures, except that they typically involve a smaller group of people and are naturally interactive, with the relationship being less asymmetrical.
+
+#### Lecture Goals
+
+The ideal lecture transfers the greates amount of knowledge / information from lecturer to participants.
+The lecture goals are aligned to that.
+These should be:
+- getting participants to attend
+- getting participants to assimilate information
+- getting participants to be (inter)active
+- providing high-quality material before, during and after lecture
+- correlating with other stages for concerted knowledge transfer
+- exciting participants about the topic
+- building a community of participants around the topic
+
+#### Lecture Metrics
+
+- attendance
+- interaction
+- exam grades
+- participant feedback
+- content validation / approval by participants and / or outsiders
+
+#### Lecture Materials
+
+- types of materials
+- focus on interactivity
+- availability outside of class
+- dynamic
+- delivery / online materials vs offline materials
+
+#### Lecture Delivery & Structure
+
+- hooking participants in: relevance of topic, connection to other topics
+- interaction
+- dynamic content, building it step by step
+- quizzes throughout lecture: mass validation of understanding
+- rough high-level structure to be adapted to each lecturer's personality and view of things
+- conclusion
+- layered approach: top-to-bottom
+
+#### Connection to Other Stages
+
+- lecture before lab, seminar
+- demos during both lectures and labs
+- direct references between lectures and labs (forward reference from lectures to labs)
+- direct references between assignments and lectures
```

Comment:
Add that assignments should build on top of lectures and labs.

Reply:
for sure